README.md

```diff
@@ -101,3 +101,7 @@ See [LICENSE](LICENSE) for details.
 Unless you explicitly state otherwise, any contribution intentionally submitted for
 inclusion in the work by you, as defined in the Apache-2.0 license,
 shall be dual licensed as above, without any additional terms or conditions.
+
+### Contributing
```

Comment:
Can you put this above the license section below the trophy case? Ideally the file would be inlined into README.md, as it's not a lot of content.

Reply:
Done.

>Ideally the file would be inlined into README.md

I often see contributing instructions placed in separate file, so I believe its some kind of convention.